cluster/config-defaults.yaml

```diff
@@ -86,7 +86,11 @@ zmon_worker_count: "0"
 {{else if eq .Environment "production"}}
 logging_agent_enabled: "true"
 zmon_agent_replicas: "1"
+zmon_agent_mem: "500Mi"
```

Comment:
You can just define it once outside the `if`, it'd be easier to keep the settings in sync.

Reply:
done